libweb3jsonrpc/LevelDBFace.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include "AbstractDb.h"
+
+namespace leveldb
+{
+	class DB;
+}
+
+class LevelDBFace: public AbstractDb
```

Comment:
`LevelDBFace` name seems misleading to me. This is not an interface, but an implementation. `AbstractDb` is an interface.